python_modules/dagster/dagster/_cli/dev.py

```diff
@@ -57,22 +57,34 @@ def dev_command_options(f):
     default="warning",
     type=click.Choice(["critical", "error", "warning", "info", "debug"], case_sensitive=False),
 )
-@click.option("--dagit-port", "-p", help="Port to use for the Dagit UI.", required=False)
-@click.option("--dagit-host", "-h", help="Host to use for the Dagit UI.", required=False)
+@click.option(
+    "--webserver-port",
+    "--dagit-port",
+    "-p",
+    help="Port to use for the Dagster webserver.",
+    required=False,
+)
+@click.option(
+    "--webserver-host",
```

Comment:
should just be `port` and `host` i think

Reply:
ye